include/deal.II/grid/grid_generator.h

```diff
@@ -1523,6 +1523,9 @@ namespace GridGenerator
    * the triangulation that contains the cells of both triangulation and store
    * it in the third parameter. Previous content of @p result will be deleted.
    *
+   * One of the two input triangulations can also be the @p result triangulation.
+   * Beware that this triangulation will be overwritten.
```

Comment:
I think the second sentence is implied :-) If it's the result, *of course* its contents will be overwritten.